### DISCUSSION TITLE
This is Awesome!!

### DISCUSSION THREAD

**Discussion by dehoyosb** [jdb78/pytorch-forecasting#1717] [General]
Hello Jan! I just read the article about this package. I was thrilled to find that it had the temporal fusion transformer, but also that it was working with pytorch-lightning!!. I came straight to the repo to check the code and the implementation, only to find out that you improved a lot upon the code I had built on my initial implementation of the TFT!!. I'm just ecstatic!!. I would love to start contributing to this repo, adding more recipes from simpler pytorch architectures, and also work on the implementation of newer ones!

I am a firm believer that the advances that have happened in Image and NLP, can happen in time series too. We just need a package like this, that helps unify the use for everybody and solves the common issues that happen in the implementations of time series models!

For my part im gonna start testing the models, because im currently working in a time series problem that im trying to solve with neural networks. I was thinking on continue the work i started on the TFT, but i think this implementation is already way better!

Lets keep in touch!

**Comment by jdb78** [comment]
Hello Daniel,

Great to hear you like the package! There are only small changes compared to the original implementation to improve performance and account for a variety of real-world inputs.

Contributions are warmly welcome. I hope this is only the start of a journey! There is a list of features here for the next release: https://github.com/jdb78/pytorch-forecasting/projects/1 But, course, something completely different, that advances the package, would be also highly appreciated.

Always happy to connect on LinkedIn,
Jan

**Comment by julioasotodv** [comment]
I have just discovered this, and I have to agree with @dehoyosb. This library looks great. Kudos for the implementation of TFT (it's probably been a ton of work). In fact, I though about starting to develop a similar lib a couple of months ago (but never ended up doing it).

Thanks a ton!
